.github/workflows/attach-static-libs.yaml

```
@@ -47,8 +47,7 @@ jobs:
       - uses: Swatinem/rust-cache@v2
 
```

This will take a lot longer unless you use the correct cache!
```suggestion
with:
          save-if: "false"
          shared-key: "ethhash"
```

---

Why use `save-if: "false"` this will never write to the cache? I was meaning to ask why this is used throughout and it seems to have something to do with `default-branch-cache.yaml`, but this also uses `save-if: "false"`.

I am reading the documentation for it here, but have not tested explicitly https://github.com/Swatinem/rust-cache

---

Answered in slack. Basically we have two caches, and we want them to contain only the results of the build, not all the subsequent steps including this one.

---

Discussed offline - `attach-static-libs.yaml` will use its own cache and be separate from the cache used in `ci.yaml`.

This will default to use the `jobid` as the shared key for this action and it does not overlap with the `build` shared key in `ci.yaml`, so this is what we want.